pandas/tests/frame/indexing/test_indexing.py

```diff
@@ -1599,6 +1599,14 @@ def test_reindex_methods_nearest_special(self):
         actual = df.reindex(target, method="nearest", tolerance=[0.5, 0.01, 0.4, 0.1])
         tm.assert_frame_equal(expected, actual)
 
+    def test_reindex_nearest_tz(self):
```

Comment:
Thanks!

Can you add the issue number number under the test? So
```
def test_reindex_nearest_tz(self):
    # GH26683
```

(many old tests didn't do this, but it seems to be preferred for new ones)